app/models/arbitrary_assignment_creation.rb

```diff
@@ -43,10 +41,10 @@ def save_new_assignment!
     save!
   end
 
-  def supersede!(now = Time.zone.now)
```

Comment:
was this optional arg just never used?

Reply:
yeah, i couldn't find any use. i'm guessing it was meant for documentation purposes.